pkg/command/service/common.go

```diff
@@ -53,6 +53,14 @@ type measureArgs struct {
 }
 
 type measureResult struct {
+	Sums         sums `json:"-"`
```

Comment:
Is `measureResult` a struct to expose make public?

Reply:
Since all its fields are public... just asking.

Reply:
All its fields are public because want to convert them to JSON. However `measureResult` is just privately needed.
May I make it public at above case? Could you pls give some advise? Thanks!